README.md

```diff
@@ -509,8 +509,29 @@ terragrunt = {
 The `remote_state` block supports all the same [backend types](https://www.terraform.io/docs/backends/types/index.html)
 as Terraform. The next time you run `terragrunt`, it will automatically configure all the settings in the
 `remote_state.config` block, if they aren't configured already, by calling [terraform
-init](https://www.terraform.io/docs/commands/init.html). The config options `s3_bucket_tags` and 
-`dynamodb_table_tags` are only valid for backend `s3`. They are used by terragrunt and are **not** passed on to 
+init](https://www.terraform.io/docs/commands/init.html). 
+
+For backend `s3`, the following config options can be used for S3-compatible object stores, as necessary:
+
+```hcl
+remote_state {
+  # ...
+
+  skip_bucket_versioning = true # use only if the object store does not support versioning
+
+  skip_region_validation      = true
```

Comment:
@brikis98 Let me know what you think about what I have documented here from this line down. Turns out Terraform docs are [stale](https://www.terraform.io/docs/backends/types/s3.html) on this and you have to read the code to get the currently available options.